BraveUI/SwiftUI/TableCellButtonStyle.swift

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import SwiftUI
+
+/// A button style that mimics a table cell being highlighted in the background
+public struct TableCellButtonStyle: ButtonStyle {
+  @Environment(\.colorScheme) private var colorScheme: ColorScheme
```

Comment:
Can we put the `private` on a new line?

So:
```swift
@Environment(\.colorScheme)
private var colorScheme: ColorScheme
```

Not sure what you guys prefer but I think it is cleaner. We do it for `@objc`?

Reply:
In all of Apple's docs/usage, it is always same-line with exceptions for some of the longer more complex property wrappers that take multiple arguments like `FetchRequest`